src/app/pages/home/components/action-button.tsx

```diff
@@ -10,9 +11,12 @@ interface ActionButtonProps extends React.ComponentProps<typeof LeatherButton> {
 }
 
 export function ActionButton({ icon, label, ...rest }: ActionButtonProps) {
+  const isBreakpointSm = useViewportMinWidth('sm');
+  const variant = isBreakpointSm ? 'ghost' : 'ghost-secondary';
+
```

Comment:
You can use panda to set this rather than having to use the hook

```
variant={{ base: 'ghost', md: 'ghost-secondary' }}
```

Reply:
seems variant prop doesn't support it
![Screenshot 2024-01-08 at 12 27 47](https://github.com/leather-wallet/extension/assets/46521087/d70463bf-06a1-4049-892c-d5bf1e965f95)

Reply:
🤔 thats odd.

I checked further and maybe we don't even need this variant at all? 

We can change the color of 'ghost' in the recipe based on the screen size? 

```
 ghost: {
        _hover: { bg: { base: 'brown.2', md: 'brown.4' } },
        ...
      },
```

Reply:
it will affect all "ghost" buttons, now sure if this is right direction here

Reply:
It looks like you are adding a new variant so that we can change the hover colour on small screens?

So we can:
- add the new variant as you did but pass it as a prop and don't use the hook
- change the hover for all ghost on small screen

You are affecting all ghost buttons anyway as if the viewport it small you switch them to another variant which is the same but has a different hover colour

Reply:
> add the new variant as you did but pass it as a prop and don't use the hook
hm, how can I can get rid of hook? if I pass variant as prop I'll need this hook in parent component, no?

> You are affecting all ghost buttons anyway
I here affect only action btns on home screen: send/recieve/buy/swap

if I change ghost variant directly it will affect all these ghost btns
![Screenshot 2024-01-08 at 15 54 46](https://github.com/leather-wallet/extension/assets/46521087/208fa190-c948-4deb-9b38-c90424c8a8f3)